02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,32 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <body style="background-color: aqua"> </p>
+  <title>Andrews Webpage bio</title>
 </head>
 <body>
+<h1>ANDREW ROBERT FLATER</h1>
```

Comment:
Indentation: All child elements of body element need to be indented by 2 spaces.